onnxscript/rewriter/pattern.py

```diff
@@ -472,15 +479,20 @@ def __init__(
         attributes: dict[str, AttrPattern],
         outputs: Sequence[str | None],
```

Comment:
```suggestion
        outputs: Sequence[str | None],
        *,
```

Consider making these options kw only for robustness